setup/tinyHIPPOstart

```diff
@@ -10,13 +10,25 @@
 opkg update
 
 # Make sure Python3, Pip, misc packages are installed
-opkg install python3
-opkg install python3-pip
 pip3 install scapy
 pip3 install dataclasses
+pip3 install requests
 
 # set config path environment variable
-export IOT_IDS_CONFIGPATH = "/etc/tinyHIPPO/config.json"
+export IOT_IDS_CONFIGPATH='/etc/tinyHIPPO/config.json'
+
+# Move the dashboard files
+# TODO: this is a hack to get tinyHIPPO running for the midterm progress report
+cp -r /etc/tinyHIPPO/src/dashboard/luasrc/controller/ids-priv /usr/lib/lua/luci/controller
+cp -r /etc/tinyHIPPO/src/dashboard/luasrc/view/ids-priv /usr/lib/lua/luci/view
+
+# Restart the webserver
+uci set luci.ccache.enable=0
+uci commit luci
+/etc/init.d/uhttpd restart
+
+# Create the alert_collection.json file
+cd /etc/tinyHIPPO/ && touch alert_collection.json
```

Comment:
Can this just be touch /etc/tinyHIPPO/alert_collection.json?

Reply:
Yes it can. Just fixed it. Will be pushing commit soon.